client/src/ui/components/quest/QuestInfo.tsx

```
@@ -33,18 +34,43 @@ export const QuestInfo = ({ quest, entityId }: { quest: Quest; entityId: ID }) =
+      </div>
+
+      {skipQuest && (
+        <>
+          <div className="flex justify-center items-baseline">
+            <p className="mr-2">Are you sure ?</p>
+            <Button variant={"primary"} isLoading={isLoading} onClick={handleAllClaims}>
+              Confirm
+            </Button>
+          </div>
+        </>
+      )}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)